pkg/distributor/otel_test.go

```diff
@@ -50,41 +49,16 @@ func TestOTelMetricsToTimeSeries(t *testing.T) {
 	}, []string{tenantID, "group"})
 	resourceAttrs := map[string]string{
 		"service.name":        "service name",
+		"service.namespace":   "service namespace",
 		"service.instance.id": "service ID",
 		"existent-attr":       "resource value",
 		// This one is for testing conflict with metric attribute.
-		"metric-attr": "resource value",
+		"metric-attr": "resource value conflict",
```

Comment:
Is there any practical reason for changing the resource attribute values? I mean, the conflicts are on the attribute names, so I don't think the values need to signal any conflict? The "resource value" part already takes care of signaling that the value comes from a resource attribute.

I also think it ends up giving the wrong impression in test expectations, as it looks as if there's a conflict in `target_info` when there is none.

Reply:
Ack, reverted that bit, it wasn't obvious to me what was going on, my change was making it a bit more explicit, but I see your point.

Reply:
Thanks, that's the rationale I suspected :)